.github/workflows/build.yml

```diff
@@ -31,6 +31,13 @@ jobs:
     - name: Smoke test (Mac, Windows)
       run: npm run test
       if: runner.os != 'Linux'
+    - name: Upload coverage reports to Codecov
+      uses: codecov/codecov-action@def81595a67c83cbb0a42ba9b624b97f4c7deb3d #v3.1.4
+      env:
```

Comment:
In [here](https://github.com/codecov/codecov-action#usage) it suggests putting these options in a `with` section, rather than an `env` section (also you'd set `token` instead of `CODECOV_TOKEN`). Do you reckon we should follow that instead?

Reply:
good point, I missed it. Thanks, I will change it soon.